docs/mockups/followup.html

```diff
@@ -310,10 +319,10 @@ <h3>
      </div>
      <div class="row">
       <div class="col-md-4">
-       <label class="visible-xs visible-sm required" for="siopss-c">
+       <label class="visible-xs visible-sm required" for="">
```

Comment:
I'm curious why you changed the for attribute to the empty string, rather than just removing it.

Reply:
That wasn't the intent. The Jinja macro always puts in a for attribute even when there is no value for the argument. I'll change that to just omit the attribute when there is no value passed.

Reply:
Makes sense.